src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/service/UserService.kt

```diff
@@ -369,7 +369,8 @@ class UserService(
   }
 
   fun lockUser(username: String) {
```

Comment:
This is really difficult to test (other than in dev) as locally the lockUser/unlock stored procedure works fine. Can test properly once in dev